processing/src/main/java/org/apache/carbondata/processing/newflow/sort/SortStepRowUtil.java

```diff
@@ -67,8 +67,8 @@
           Object value = data[allCount];
           if (null != value) {
             if (measureDataType[i] == DataType.DECIMAL) {
-              BigDecimal decimal = (BigDecimal) value;
-              measures[index++] = DataTypeUtil.bigDecimalToByte(decimal);
+              byte [] bytes =(byte[]) value;
+              measures[index++] = bytes;
```

Comment:
I faced the class cast exception when I load data byte[] can not be cast to big decimal